wordpress-mobile/WordPress-Android
# My Profile Fragment

Fixes #4416. Converts `MyProfileActivity` to fragment for tablet refresh project. I tried not to touch anything unrelated to the fragment conversion. The only thing I am not so happy about is that we are calling `((Callback) getActivity()).onSuccessfulInput()` inside the `ProfileInputDialogFragment` which means we need to handle this inside the activity and pass it to the fragment. Any ideas to improve that would be welcome!

To test:
* Go into My Profile page and change your information, check remote to see if the changes are reflected
* Change your display name and go back to the Me page and check if the changes are reflected there (under the gravatar)

/cc @astralbodies 

Needs review: @hypest
